src/libexpr/eval.cc

```diff
@@ -1709,7 +1710,8 @@ void EvalState::callFunction(Value & fun, size_t nrArgs, Value * * args, Value &
                 /* We have all the arguments, so call the primop with
                    the previous and new arguments. */
 
-                Value * vArgs[arity];
+                assert(arity < 64);
```

Comment:
Should we maybe have a constant for this 64? I see it used a lot here.

That, or a comment explaining where it came from wherever it's used.

Reply:
Done. See `maxPrimOpArity`.